assets/js/components/setup/setup-wrapper.js

```diff
@@ -96,77 +68,79 @@ class SetupWrapper extends Component {
 		delay( function() {
 			global.location.replace( redirectURL );
 		}, 500, 'later' );
+	}, [ moduleSlug ] );
+
+	if ( ! module ) {
+		return null;
 	}
 
-	render() {
-		const { currentModule } = this.state;
-		const setupModule = SetupWrapper.loadSetupModule( currentModule );
-		const settingsPageURL = getSiteKitAdminURL(
-			'googlesitekit-settings',
-			{}
-		);
+	const { setupComponent: SetupComponent } = module;
```

Comment:
Based on my above comment, we could either remove the `DefaultModuleSetup` component entirely (it strictly isn't needed), or we could use it here internally to render it if `SetupComponent` is `undefined`. Since this screen would in the future never be linked to if there is no `setupComponent`, it may also be okay just rendering nothing in that case.